Enable TLS Fingerprinting configuration (JA3/JA4) (#7372)
projectcontour/contour: site/content/docs/main/config/access-logging.md

- `contour_config_namespace`
- `contour_config_name`

### Logging TLS Fingerprints (JA3/JA4)

> When [JA3/JA4 TLS fingerprinting is enabled](../tls-fingerprinting) in the Contour configuration, fingerprints can be logged in access logs.

Reviewer: The link is not valid, use this instead

```suggestion
When JA3/JA4 TLS fingerprinting is enabled in [configuration file](../configuration#tls-fingerprint) or [ContourConfiguration resource](api-reference.html#projectcontour.io/v1alpha1.TLSFingerprint), fingerprints can be logged in access logs.
```

The description is also missing for contour configuration file alternative. You can add this:

```diff
diff --git a/site/content/docs/main/configuration.md b/site/content/docs/main/configuration.md
index 7fd5b5c26..85b65b7d3 100644
--- a/site/content/docs/main/configuration.md
+++ b/site/content/docs/main/configuration.md
@@ -117,6 +117,7 @@ Contour should provision TLS hosts.
 | fallback-certificate     |          |                                                                                                                   | [Fallback certificate configuration](#fallback-certificate).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 | envoy-client-certificate |          |                                                                                                                   | [Client certificate configuration for Envoy](#envoy-client-certificate).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 | cipher-suites            | []string | See [config package documentation](https://pkg.go.dev/github.com/projectcontour/contour/pkg/config#pkg-variables) | This field specifies the TLS ciphers to be supported by TLS listeners when negotiating TLS 1.2. This parameter should only be used by advanced users. Note that this is ignored when TLS 1.3 is in use. The set of ciphers that are allowed is a superset of those supported by default in stock, non-FIPS Envoy builds and FIPS builds as specified [here](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto#envoy-v3-api-field-extensions-transport-sockets-tls-v3-tlsparameters-cipher-suites). Custom ciphers not accepted by Envoy in a standard build are not supported. |
+| fingerprint              |          |                                                                                                                   | [TLS fingerprinting configuration](#tls-fingerprint).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |

 ### Upstream TLS Configuration

@@ -144,6 +145,16 @@ The Upstream TLS configuration block can be used to configure default values for
 | namespace  | string | `""`    | This field specifies the namespace of the Kubernetes secret to use as the client certificate and private key when establishing TLS connections to the backend service. |

+### TLS Fingerprint
+
+When enabled, TLS fingerprints are computed for HTTPS listeners and can be logged in access logs. See [Logging TLS Fingerprints](config/access-logging#logging-tls-fingerprints-ja3ja4) for more information.
+
+| Field Name | Type    | Default | Description                          |
+| ---------- | ------- | ------- | ------------------------------------ |
+| ja3        | boolean | `false` | Enables JA3 TLS fingerprinting. |
+| ja4        | boolean | `false` | Enables JA4 TLS fingerprinting. |
+
+
 ### Timeout Configuration

 The timeout configuration block can be used to configure various timeouts for the proxies. All fields are optional; Contour/Envoy defaults apply if a field is not specified.
```
